SECURITY.md

```diff
@@ -0,0 +1,12 @@
+# Security Policy
+Sail project takes security seriously and encourages users to report security concerns.
+
+If you run a security scan on Sail operator software that automatically generates a list of potential vulnerabilities and would like to report this security scan report to the Sail operator team, we ask that you first verify that your scan is correctly validating the latest release and that the list of results is valid, contains no duplicates, and the reported vulnerabilities truly affect project. Security report investigation often takes priority over scheduled work and can be time consuming for the Sail maintainers to research and validate. So, please verify that your submitted report accurately reflects the Sail software being scanned, and that the reported security issue(s) actually affect Sail or one of its dependencies.
```

Comment:
I think we should use the spelling `sail-operator` everywhere

Reply:
Done